.travis.yml

```diff
@@ -9,6 +9,9 @@ rvm:
   - 2.1.2
   - 2.2.3
 
+env:
+  - DEBUG=1 CI_NO_ANDROID_RUNTIME=1
```

Comment:
> DEBUG=1 on Travis

We should probably not commit this because Travis is very picky about the number of lines that are logged.

If you pass a certain limit, the build will error.